src/App.tsx

```diff
@@ -4,6 +4,16 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [guery, setQuery] = React.useState<string>('');
+
+  const isMovieMatchQuery = (movieProperty: string) => {
+    return movieProperty.toLowerCase().includes(guery.toLowerCase().trim());
+  };
+
+  const visibleMovies = moviesFromServer.filter(movie => (
```

Comment:
You can also do like that
```suggestion
  const visibleMovies = moviesFromServer.filter(({ title, description })=> (
```